front/src/modules/trainschedule/components/ManageTrainSchedule/Map.tsx

```diff
@@ -385,7 +385,7 @@ const Map = ({
               layerOrder={LAYER_GROUPS_ORDER[LAYERS.LINE_SEARCH.GROUP]}
               infraID={infraID}
             />
-            <RenderPopup pathProperties={pathProperties} />
+            {!showStdcmAssets && <RenderPopup pathProperties={pathProperties} />}
```

Comment:
Hm, why this change? Is this fixing a bug?

Reply:
This popup is displayed only if I click on the map. This feature is only available in the EEx, and inside the component, I use the ManageTrainScheduleContext.
If this component renders in Stdcm, then the app crashed because there is no ManageTrainScheduleContextProvider.

Reply:
Aha, makes sense!